be2-scala/src/main/scala/ch/epfl/pop/pubsub/ClientActor.scala

```diff
@@ -25,6 +25,10 @@ final case class ClientActor(mediator: ActorRef, connectionMediatorRef: ActorRef
     case ClientAnswer(graphMessage) => wsHandle.fold(())(_ ! graphMessage)
   }
 
+  // Tell connectionMediator we are online
+  if (isServer)
+    connectionMediatorRef ! ConnectionMediator.NewServerConnected(self)
```

Comment:
Nitpick but this could be put above `private def messageWsHandle(event: ClientActorMessage): Unit = ...` to avoid having a "constructor instruction" appear out of nowhere between two method definitions.

Reply:
Fixed !